packages/ui-components/src/numeric-input/numeric-input.spec.tsx

```diff
@@ -95,6 +105,17 @@ describe('Numeric Input', () => {
         expect(input).toHaveValue('5');
     });
 
+    it('should erase . if decimal value was not fully entered', async () => {
+        const { getByRole } = render(<MockControlledNumericInput />);
+        const input = getByRole('textbox', { hidden: true });
+
+        await userEvent.type(input, '10.', { delay: 1 });
+        expect(input).toHaveValue('10.');
+
+        userEvent.keyboard('{backspace}');
```

Comment:
Missing await?

Reply:
I had it initially but ide corrected saying it is not needed here